package.json

```diff
@@ -2,6 +2,7 @@
   "name": "did-cli",
   "version": "0.4.2-0",
   "description": "A program for managing Decentralized Identifiers (DIDs).",
+  "license": "SEE LICENSE IN LICENSE.md",  
```

Comment:
It's just 'LICENSE' in this project, not a md file.

Reply:
Or in this case, perhaps it should just be the proper BSD tag rather than this note.

Reply:
Fixed.